tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>same here, can we avoid the <code>create</code>  method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Creating a new interval list requires 2 arguments:  the list of time ranges and observation timestamps.  If you know of a way to pass more than one argument to the setitem method of a class, then I could avoid a dedicated create method here and elsewhere.  Should I make setitem take a dictionary of parameters?</p><pre class="ql-syntax" spellcheck="false">ob.intervals["good"] = {
    "ranges": [(1.5, 3.5), (4.5, 6.), (7., 8.5)], 
    "times": ob.shared["times"]
}
</pre><p>I could do that and put a check in setitem for such things.  Do you think that is more clear than having a <code>create()</code>  method?</p>

Reply:
<p>This is the last remaining comment from your last round of review.  I have not yet found an easier way to create intervals on assignment for a single value on the RHS.  I'll leave this comment open, but I'm going to move on to porting more operators and discussion of the instrument classes.</p>